content/blog/2021/06/kwctl-intro-for-kubernetes-administrators.md

```diff
@@ -0,0 +1,307 @@
+    is going to be changed
+
+Hence, this JSONPatch adds "AUDIT_READ" to the original Pod object, at
+`/spec/containers/0/securityContext/capabilities/add/2` location.
+
+Let's take a closer look at the JSON Pointer:
+
+  * **`/spec/containers`**: this references the `containers` section of the Pod `spec`
+  * **`/0`**: since `spec.containers` is an array, this fragment of
+    the path points to the first element of it. Note well, arrays elements
+    are referenced using [zero-based numbering](https://en.wikipedia.org/wiki/Zero-based_numbering)
+  * **`/securityContext/capabilities/add`**: this navigates into the `securityContext`
+    of the container, then into the `capabilities` object and finally into the
+    `add` section. This is the place were container capabilities to be added
+    are specified
+  * **`/2`**: the original container has already two capabilities to be added,
+    `NET_ADMIN` and `SYS_TIME`. Using zero-based numbering, `/2` points to
+    the third entry of the array.
+
+The policy is working as expected, it leads to the creation of a Pod with
+a container that has the `NET_ADMIN`, `SYS_TIME`, `AUDIT_READ` capabilities
+added.
+
+## Deploying the policy
+
+After more experiments with `kwctl run`, Alice wants to deploy the policy
+inside of a Kubernetes cluster.
+
+Kubewarden policies are defined using the
+[ClusterAdmissionPolicy](https://github.com/kubewarden/kubewarden-controller/blob/a74144e2c2f69dd96a7317feb44719a03677a885/docs/crds/README.asciidoc),
+a Custom Resource provided by Kubewarden.
+
+Alice can quickly scaffold the YAML file defining this resource using kwctl:
+
+```shell
+$ kwctl manifest -t ClusterAdmissionPolicy registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4
+```
+
+The command will produce the following output:
+
+```yaml
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: generated-policy
+spec:
+  module: "registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4"
+  settings: {}
+  rules:
+    - apiGroups:
+        - ""
+      apiVersions:
+        - v1
+      resources:
+        - pods
+      operations:
+        - CREATE
+  mutating: true
+```
+
+kwctl just saved quite some time to Alice 😊
+
+## Mirroring policies
+
+Kubewarden policies are distributed via OCI container registries, the very
+same pieces of infrastructure already used to distribute container images.
+
+Alice doesn't want her cluster to pull the Kubewarden policy straight from
+`ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4`, she rather prefers to
+fetch it from a local registry she controls.
+
+This can be done using the following commands:
+
+```shell
+$ kwctl tag registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4 registry://registry.alice.corp.lan/policies/psp-capabilities:v0.1.4
+$ kwctl push registry://registry.alice.corp.lan/policies/psp-capabilities:v0.1.4
+```
+
+The policy is now ready to be consumed by the local Kubernetes cluster 🎉
```

Comment:
Where's the step to actually deploy the policy ?

Reply:
I've expanded a bit on that. I wouldn't write more on that because these are "common" k8s operations